task_templates/README.md

```diff
@@ -1,50 +1,70 @@
 # Templates for Custom Tasks
+
 In this directory are three folders - estimators, transforms, and pipelines
 
 * estimators - meant to be used as the prediction step in a blueprint
 * transforms - meant to be used as a preprocessing step
```

Comment:
instead of 

"transforms - meant to be used as a preprocessing step"

let's say

"transforms - meant to be used as a preprocessing step in a blueprint"